Task_24 added

Anton-Gursky opened the pull request.

Commit:
Task_24 added

DHrytsuk commented on src/main/java/org/example/Order.java:
```diff
@@ -0,0 +1,35 @@
+    private int quantity;
+    private int price;
+
+    public Order() {
```
You could skip adding empty constructor as it's added by default inside java compiler, when there are no other constructors

DHrytsuk commented on src/main/java/org/example/Order.java:
```diff
@@ -0,0 +1,35 @@
+    public Order() {
+    }
+
+    public String getOrderName() {
```
Minor:
in general, it's better to place getter and setter for the same field together (like it does IDEA when auto-creating)

DHrytsuk commented on src/main/java/org/example/Main.java:
```diff
@@ -0,0 +1,41 @@
+public class Main
+{
+    private static final Logger log = Logger.getLogger(Main.class);
+    private static final Logger orderLogger = Logger.getLogger("SECOND_LOGGER");
```
It's kind of hack to write file with a logger 😄 
+1 for respect for such approach, but let's use general approach)

DHrytsuk commented on src/main/java/org/example/Main.java:
```diff
@@ -0,0 +1,41 @@
+        System.out.println("Введите цену товара в диапазоне от 10 до 100:");
+        order.setPrice(scanner.nextInt());
+
+        while (order.getPrice() < 10 || order.getPrice() > 100){
```
👍

DHrytsuk commented on src/main/java/org/example/Main.java:
```diff
@@ -0,0 +1,41 @@
+
+    public static void main( String[] args )
+    {
+        Order order = new Order();
```
Let's wrap this code into the loop to add ability to add more than 1 order during program execution

DHrytsuk commented on src/test/java/org/example/MainTest.java:
```diff
@@ -7,15 +7,15 @@
  */
-public class AppTest 
+public class MainTest
     extends TestCase
```
No need to extend something

DHrytsuk commented on src/main/java/org/example/Main.java:
```diff
@@ -0,0 +1,41 @@
+    private static final Logger log = Logger.getLogger(Main.class);
+    private static final Logger orderLogger = Logger.getLogger("SECOND_LOGGER");
+
+    public static void main( String[] args )
```
Let's split your main method into several (eg. method to get info, method to validate price, method to write file) and test some of them - it will be easier

Commit:
modified
1. The possibility to add more than one order added,
2. A few methods are added.
